scripts/functions/selector_interpreters

```diff
@@ -272,6 +272,17 @@ __rvm_truffleruby_set_rvm_ruby_url()
 
     rvm_ruby_package_file="${rvm_ruby_package_name}-${platform}"
     rvm_ruby_url="${rvm_ruby_repo_url:-https://github.com/ruby/truffleruby-dev-builder/releases/latest/download/${rvm_ruby_package_file}.tar.gz}"
+  elif [ "$truffleruby_version" = "23.0.0" ]; then
+    rvm_ruby_package_file="${rvm_ruby_package_name}-${platform}-${arch}"
+    truffleruby_artifact_id=""
+    case "${platform}-${arch}" in
+      linux-amd64)    truffleruby_artifact_id="FD4AB182EA4CEDFDE0531518000AF13E" ;;
```

Comment:
These URLs are not pretty, but they are stable and should keep working forever.
We're working on getting a stable URL format for them, similar to `truffleruby+graalvm-23.0.0` in the other file.